src/routes/alerts/entities/alerts.entity.ts

```diff
@@ -0,0 +1,34 @@
+export type AlertLog = {
+  address: string;
+  topics: Array<string>;
+  data: string;
+};
+
+export type AlertTransaction = {
+  network: string;
+  block_hash: string;
+  block_number: number;
+  hash: string;
+  from: string;
+  to: string;
+  logs: Array<AlertLog>;
+  input: string;
+  value: string;
+  nonce: string;
+  gas: string;
+  gas_used: string;
+  cumulative_gas_used: string;
+  gas_price: string;
+  gas_tip_cap: string;
+  gas_fee_cap: string;
+};
+
+export enum AlertEventType {
+  ALERT = 'ALERT',
+}
```

Comment:
Personally I find it strange that an `AlertEventType` would have as a type `ALERT`.

I'd probably use `EventType` here wdyt?

Reply:
Very true. I've adjusted it in ffbf4a04e7995592f9eeb14a12cfb84342b54672.